changelog/updates/2022-12-12-portage-update.md

```diff
@@ -0,0 +1,3 @@
+- portage ([3.0.41](https://gitweb.gentoo.org/proj/portage.git/tree/NEWS?h=portage-3.0.41))
+- file ([5.43](https://mailman.astron.com/pipermail/file/2022-September/000857.html))
```

Comment:
Maybe add prefix `SDK: ` to both portage and file?

Reply:
I tend to not add the entries for software that is only in SDK unless it's kind-of-a-major piece of software. In this case, both `portage` and `file` are also a part of the developer container (thus the `(DEV)` annotation in the description above), that's why I added the changelog without any prefixes.

We haven't yet written down what updates should go into the changelog, so my loosely rule so far was that I log updates of important software for production images and developer containers, and same for SDK with the `SDK: ` prefix.